### ISSUE TITLE
Issue with UnusedUsesSniff in latest version

### ISSUE THREAD

**Issue by marcingy** [slevomat/coding-standard#293]
We have a case where by phpunit annotations are flagging errors incorrectly with case

We have a use statement as follows
`
use Aws\Sns;
`
And then the following annotation
`
* @group SNS
`

And this gives the error
`
 15 | ERROR | Case of reference name "SNS" and use statement "Sns" do not match.
`
There is also a similar issue with @exceptedExceptionMessage. 

I wonder if the fix for this long term to allow a blacklist of annotations not to sniff?

**Comment by kukulich** [comment]
@marcingy There's an option (only in master now) to ignore annotation names. Could you prepare PR with new option `ignoreAnnotations` to ignore the whole annotation line?

**Comment by marcingy** [comment]
Created #294 for this.

**Comment by kukulich** [comment]
No, you didnt understand it.

**Comment by lcobucci** [comment]
@kukulich I do have an issue here (running `4.5.0`):

```php
<?php
declare(strict_types=1);

namespace Blah;

use const OPENSSL_ALGO_SHA256;
use const OPENSSL_ALGO_SHA384;
use const OPENSSL_ALGO_SHA512;

final class Testing
{
    public const LENGTH = [
        OPENSSL_ALGO_SHA256 => 64,
        OPENSSL_ALGO_SHA384 => 96,
        OPENSSL_ALGO_SHA512 => 132,
    ];
}

```

```sh
FILE: Testing.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 7 | ERROR | [x] Type OPENSSL_ALGO_SHA384 is not used in this file.
 8 | ERROR | [x] Type OPENSSL_ALGO_SHA512 is not used in this file.
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

Should I open a new issue?

**Comment by kukulich** [comment]
@lcobucci Yes, please.

**Comment by kukulich** [comment]
@lcobucci Fixed in master, I'll relase 4.5.1 this evening or tomorrow it anything else pop out.

**Comment by lcobucci** [comment]
@kukulich awesome! Thanks a lot =)

**Comment by marcingy** [comment]
I have had a chance to update and I am still getting errors around line 200 as the annotations get flagged for case violations there when they are totally fine as they are simply strings that happen to match a class name.

**Comment by kukulich** [comment]
New option implemented in https://github.com/slevomat/coding-standard/commit/f98302fc594ae9ed5a0f2fcea6b8fc627248223d
